release-notes/opendistro-for-elasticsearch.security-kibana.release-notes-1.7.0.0.md

```diff
@@ -0,0 +1,3 @@
+##2020-05-05, Version 1.7.0.0
+
+- Support for Elasticsearch 7.6.1
```

Comment:
Why there are 2 versions adding support for 7.6.1?

Reply:
Because after ODFE 1.6.0.0 release, we decided to change the upcoming release to a minor release. This means the new version will be renamed from 1.6.1 to 1.7.0.  As we didn't changed anything after 1.6.0, it remains same. Infra team created 1.7 branch based on the existing 1.6 branch.